internals/schemas/app.schema.json

```diff
@@ -394,6 +394,9 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "crs": {
+          "type": "string"
+        },
```

Comment:
please add a description and the three types of strings that are possible

Reply:
Added this 👍

Reply:
thanks!